emission/net/ext_service/transit_matching/match_stops.py

```diff
@@ -3,24 +3,23 @@
 import requests
 import attrdict as ad
 import itertools
-import copy
+import os
 import time
 
 try:
-    config_file = open('conf/net/ext_service/overpass_server.json')
+    OVERPASS_KEY = os.environ.get("OVERPASS_KEY")
```

Comment:
nit: I would rename this to include GEOFABRIK in the name (e.g. `GEOFABRIK_OVERPASS_KEY`), since there might be other commercial providers of OSM data.